core/src/main/java/com/netflix/msl/MslError.java

```diff
@@ -136,6 +136,7 @@
     public static final MslError USERIDTOKEN_USERDATA_PARSE_ERROR = new MslError(2018, ResponseCode.USER_REAUTH, "Error parsing user ID token user data.");
     public static final MslError USERIDTOKEN_REVOKED = new MslError(2019, ResponseCode.USER_REAUTH, "User ID token is revoked.");
     public static final MslError USERIDTOKEN_REJECTED_BY_APP = new MslError(2020, ResponseCode.USERDATA_REAUTH, "User ID token is rejected by the application.");
+    public static final MslError USERIDTOKEN_REVOKE_CHECK_ERROR = new MslError(2021, ResponseCode.USER_REAUTH, "User ID token could not be checked for revocation.");
```

Comment:
Since this MslError is only used during user authentication, rather than user ID token processing, I think it's probably best part of the 5 User Authentication error set. You may notice there are some user ID token related MslErrors defined under 5 User Authentication already, like USERAUTH_USERIDTOKEN_MISSING or USERAUTH_USERIDTOKEN_INVALID.

If there is an error checking for revocation, the response code probably needs to be USERDATA_REAUTH. 2 User ID Token has several USER_REAUTH response codes, but 5 User Authentication operations almost always need USERDATA_REAUTH.